kubetest2/kubetest2-gke/deployer/commandutils.go

```diff
@@ -29,6 +29,22 @@ func (d *deployer) prepareGcpIfNeeded() error {
 	// TODO(RonWeber): This is an almost direct copy/paste from kubetest's prepareGcp()
 	// It badly needs refactored.
 
+	var endpoint string
+	switch env := d.environment; {
+	case env == "test":
+		endpoint = "https://test-container.sandbox.googleapis.com/"
+	case env == "staging":
+		endpoint = "https://staging-container.sandbox.googleapis.com/"
+	case env == "staging2":
+		endpoint = "https://staging2-container.sandbox.googleapis.com/"
+	case env == "prod":
+		endpoint = "https://container.googleapis.com/"
+	case urlRe.MatchString(env):
+		endpoint = env
+	default:
+		return fmt.Errorf("--gke-environment must be one of {test,staging,prod} or match %v, found %q", urlRe, env)
+	}
```

Comment:
copied from kubetest/gke.go

https://github.com/kubernetes/test-infra/blob/f5dd3ee0e9db3b4c67073815b399aa66a1620d2e/kubetest/gke.go#L192-L206

Reply:
my personal GCP account can only access production endpoint, this is required outside of the prow platform